src/storage/leveldb/store.ts

```diff
@@ -63,6 +68,7 @@ export default class LevelDBStore implements IStore {
     await this.utxoIndex.clear();
     await this.walletIndex.clear();
     await this.tokenIndex.clear();
+    await this.nanoContractIndex.clear();
     await this.close();
   }
 
```

Comment:
Don't you need to add `await this.nanoContractIndex.validate();` to the method below?

Reply:
Fixed at https://github.com/HathorNetwork/hathor-wallet-lib/pull/614/commits/f7f228e35c65a12e4ab9eade924898eb8c63348b